pyproject.toml

```diff
@@ -60,6 +60,7 @@ markers = [
 
 [tool.mypy]
 # Import discovery
+mypy_path = "stubs"
```

Comment:
https://github.com/microsoft/pyright/issues/2514#issuecomment-955066685

`stubPath = "stubs"` for pyright or rename the folder to `typings`.

Reply:
Probably also need to append "stubs" to `include` to make sure that pyright also checks `stubs` itself for any errors/suggestions.

Reply:
done for mypy. what do I need to change for pyright?

Reply:
I think the following should work
```
include = ["pandas", "stubs"]
stubPath = "stubs"
```

You can probably also remove the pyright file-based ignore comments.

Reply:
If it is setup correctly, pyright will complain about the incomplete `__getattr__` (add a file-based ignore comment).